### DISCUSSION TITLE
NSClient++ on RHEL 7

### DISCUSSION THREAD

**Discussion by sysshob** [mickem/nscp#791] [Q&A]
Hi,

On https://nsclient.org/docs/ I saw Linux is one of the supported OS/platform: "Debian, Centos and Ubuntu (and possibly others as well)".
I need to run NSClient++ on RHEL 7 (Red Hat Enterprise Linux).
1. Is this OS supported?
2. Where can I download a suitable installation?
3. If an installation file isn't available, how can I compile the source code on RHEL 7 myself?
Please provide a detailed step-by-step, as I'm native to neither Linux nor C++ compilations.

Thanks

**Comment by mickem** [comment]
Build instructions and packages are provided now
